docs/blog/pydantic-2-alpha.md

```diff
@@ -0,0 +1,40 @@
+# Pydantic 2 Alpha Release
+
+<aside class="blog" markdown>
+![Samuel Colvin](/img/samuelcolvin.jpg)
+<div markdown>
+  **Samuel Colvin** &bull;&nbsp;
+  [:material-github:](https://github.com/samuelcolvin) &bull;&nbsp;
+  [:material-twitter:](https://twitter.com/samuel_colvin) &bull;&nbsp;
+  :octicons-calendar-24: April 01, 2023 &bull;&nbsp;
+  :octicons-clock-24: 5 min read
+</div>
+</aside>
+
+---
+
+We're excited to announce the first alpha release of Pydantic 2!
+
+This first Pydantic 2 alpha no April Fool's joke. After months of work, we invite you to explore the improvements we've made and give us your feedback. We look forward to hearing your thoughts and working together to improve the library.
+
+For many of you, Pydantic is already a key part of your Python toolkit and needs no introduction &mdash; we hope you'll find the improvements and additions in Pydantic 2 useful.
+
+If you're new to Pydantic: Pydantic is an open-source Python library that provides powerful data parsing and validation &mdash; including type coercion and useful error messages when typing issues arise &mdash; and settings management capabilities. See [the docs](/) for examples of Pydantic at work
+
+Note that there are still some rough edges and incomplete features, and while using the Pydantic 2 alpha releases you may experience errors. If you do encounter any issues, please [create an issue in GitHub](https://github.com/pydantic/pydantic/issues) using the `bug V2` label. This will help us to actively monitor and track errors, and to continue to improve the library’s performance.
+
+This will be the first of several upcoming alpha releases. We recommend that you use the alpha releases in a test environment and not in production. Some features are still in development, and we will continue to make changes to the API.
+
+Thank you for your support, and we look forward to your feedback.
+
+---
+
+## Headlines
+
+Here are some of the most interesting new features in the current Pydantic 2 alpha release. For background on these features, see the earlier [Pydantic V2](/blog/pydantic-v2/) Plan blog post.
+
+### Ready for experimentation
+
+### Work in progress
+
+### Under construction
```

Comment:
Obviously we still need to flesh this out — is there a plan for how we'll get this populated? (Not sure if I should be drafting something up here, or if Samuel is going to own it, or if we'll schedule a discussion, or ...?)

Reply:
I'm a bit relying you you and Samuel to draft the main points

Reply:
I'll just draw @adriangb's attention to this as well.